packages/xo-web/src/xo-app/logs/backup-ng/log-alert-body.js

```diff
@@ -351,39 +365,47 @@ export default decorate([
 
         let newLog
         log.tasks.forEach((task, key) => {
-          if (task.tasks === undefined || get(() => task.data.type) !== 'VM') {
+          if (get(() => task.data.type) !== 'VM') {
             return
           }
 
-          const subTaskWithIsFull = task.tasks.find(
-            ({ data = {} }) => data.isFull !== undefined
+          const vm = vms[task.data.id]
+          const subTaskWithIsFull = get(() =>
```

Comment:
Why this change?

Reply:
`undefined` `task.tasks` isn't handled anymore.